_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Version 1
+---
+
+We are happy to release version 1 of MDAnalysis!
+
+
+MDAnalysis 1.x is a stable legacy platform that supports 
+
+-  Python 2.7 (on Linux and MacOS only), 
+-  Python 3.5, 3.6, 3.7 and 3.8 (on Windows (32- and 64-bit), MacOS, and Linux).
```

Comment:
please check: no Windows for 2.7, right?

Reply:
Yup, 3.5+ for Windows.